registry/storage/manifeststore.go

```diff
@@ -115,6 +134,7 @@ func (ms *manifestStore) Get(ctx context.Context, dgst digest.Digest, options ..
 		return ms.schema1Handler.Unmarshal(ctx, dgst, content)
 	case 2:
 		// This can be an image manifest or a manifest list
+		// *********Proposal******: refactor to lookup the handler in the map for the media type and fallback to ocimanifest if not found
```

Comment:
What if a manifest is not in JSON? For example, an XML?

Reply:
The manifest is an interface where Payload() returns bytes, so the XML manifest handler can unmarshall to return a manifest that implements payload to return the  xml manifest bytes?